core/src/smartcontracts/wasm.rs

```diff
@@ -274,6 +274,10 @@ fn create_config() -> Result<Config> {
         .consume_fuel(true)
         .cache_config_load_default()
         .map_err(Error::Initialization)?;
+    #[cfg(feature = "wasm_profiling")]
```

Comment:
```suggestion
    #[cfg(wasm_profiling)]
```

what I would prefer is to make it a compilation flag and then snuck it in with `rustflags` instead of defining a cargo feature

Reply:
then you can just build iroha with:
`cargo build --profile profiling`

Reply:
oh, good point!

Reply:
changed, hope copiler flags in cargo.toml would be stabilized soon